### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by AbhishekkGautam**
### Home Page

quiz-app-home

### CODE REVIEW

**Review by someshtiwarii — APPROVED** [review]
good to merge

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+								<p class="card-description">Test your general knowledge about India. </p>
+								<button class="btn btn-play">Play</button>
+							</div>
+						</div>
+					</div>
+					<div class="featured-categories-card">
+						<div class="card quiz-card card-shadow">
+							<div class="card-header">
+								<img class="img-responsive" src="https://reactjs.org/logo-og.png" alt="..">
+							</div>
```
Comment (poornima-alugubelly) [inline index.html:81]: can we add a better description in alt instead of .. ?

**Review by poornima-alugubelly — APPROVED** [review]
good to go

File: index.html (hunk 1 above)
Comment (AbhishekkGautam) [reply index.html:81]: Done. Thank you

File: css/dafault.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  --primary-color: #272838;
+  --secondary-color: #3f3549;
+  --text-color: #989fce;
+  --accent-color: #c4a6e3;
+  --gray-50: #f5f5f4;
+  --gray-100: #e9e9e9;
+  --gray-300: #d6d3d1;
+  --gray-600: #57534e;
+  --gray-800: #292524;
+  --white: #fff;
```
Comment (msdeshmukh009) [inline css/dafault.css:11]: Suggestion: Follow some kind of pattern while naming color variables. Maybe use any prefix or suffix in the name.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking fine.

File: css/dafault.css (hunk 2 above)
Comment (AbhishekkGautam) [reply css/dafault.css:11]: these are the different shades of gray...from lighter (100) to darker (800)

**Pull request merged**